portfolio/src/main/webapp/src/com.js

```diff
@@ -115,25 +117,37 @@ function deleteAllCommentsFromServer() {
 }
 
 /**
- * Creates a comment card with the given comment text
+ * Creates a comment card with the given comment poster, comment text
  * and sentiment score if it is provided.
  *
  * @param {string} commentText
+ * @param {any} commentPoster
  * @param {any} sentimentScore Potential sentiment score,
  * or undefined if no score to be displayed.
  * @return {Element}
  */
-function createCommentCard(commentText, sentimentScore = undefined) {
+function createCommentCard(
+  commentText,
+  commentPoster = undefined,
+  sentimentScore = undefined,
+) {
   const cardElement = document.createElement('div');
   cardElement.classList.add('port-card');
 
+  if (typeof commentPoster === 'string') {
```

Comment:
Do you want to check truthiness instead of type here? You can decide whether or not `''` is meaningful in this case.

Reply:
I'll check truthiness -- `''` is meaningless in this case.